src/translations/en_US.js

```diff
@@ -27,6 +27,8 @@ module.exports = {
   paypalTokenizationRequestActiveError: 'PayPal payment authorization is already in progress.',
   applePayTokenizationError: 'A network error occurred while processing the Apple Pay payment. Please try again.',
   applePayActiveCardError: 'Add a supported card to your Apple Pay wallet.',
+  venmoCanceledError: 'Something went wrong. Either you cancelled the app, your browser does not support switching to the Venmo app, or Venmo is not installed on your device.',
```

Comment:
Does this wording look good to y'all?

Reply:
Nothing wrong but it feels a bit long...

Tempted to drop `Either you cancelled the app` since they would already know that they cancelled it and it probably shouldn't appear as an error?

It could be helpful to ask them to be on the latest version too?

`Something went wrong. Ensure you have the most recent version of the Venmo app installed on your device and your browser supports switching to the Venmo.`?

Reply:
:+1: